load-data-from-kafka/generated/leads/persist_client.bal

```diff
@@ -0,0 +1,104 @@
+// AUTO-GENERATED FILE. DO NOT MODIFY.
+// This file is an auto-generated file by Ballerina persistence layer for model.
+// It should not be modified by hand.
+import ballerina/jballerina.java;
```

Comment:
Let's remove the generated stuff from here and in the readme add the step to `persist generate`